src/WebApp/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "codidact.core",
+  "version": "1.0.0",
+  "description": "Codidact Platform",
+  "scripts": {
+    "tsc": "tsc",
```

Comment:
I am not super familiar with npm, but shouldn't this be `npx tsc`?

Reply:
True, I have ts installed globaly so this is why it worked for me